CHANGELOG

```diff
@@ -1,4 +1,5 @@
 == Unreleased
+- Update API version with 2022-10 release, remove API version 2021-10
```

Comment:
when this will be available in released version

Reply:
It seems to be available now. Version 12.1.0 is on pypi as of a few hours ago.

Reply:
Thanks its available now